frontend/src/v5/ui/themes/theme.ts

```diff
@@ -73,6 +73,8 @@ export const COLOR = {
 	FAVOURITE_MID: '#FDBF4E',
 	FAVOURITE_LIGHT: '#FAE59A',
 	FAVOURITE_LIGHTEST: '#FEFAEB',
+	INFO_MAIN: '#0288D1',
+	INFO_LIGHT: '#80C3E8',
```

Comment:
These are new colours  and only being used in this feature. Cant we use a colour that that's already in the palette?

Reply:
Sorry I nosed in - Can we avoid ceating additional colours for bespoke designs?

I think in this situation the info box can just use the tertiary colours in the theme.

![image](https://github.com/3drepo/3drepo.io/assets/11945337/db0024f7-2d85-4fc1-8570-adfe7e1b96c2)